index.js

```diff
@@ -145,26 +146,27 @@ export const switchToggle = {
 };
 
 export const toaster = {
-  toasterContainer:
+  container:
     'fixed fixed-ios-fix bottom-16 left-0 right-0 mx-8 sm:mx-16 z-50 pointer-events-none',
+  content: 'w-full',
   toaster:
     'grid f-grid auto-rows-auto justify-items-center justify-center mx-auto pointer-events-none',
 };
 
 export const toast = {
-  toastWrapper: 'relative overflow-hidden w-full',
+  wrapper: 'relative overflow-hidden w-full',
   toast:
     'flex group p-8 mt-16 rounded-8 border-2 w-full pointer-events-auto transition-all',
-  toastPositive: 'i-bg-$color-toast-positive-background i-border-$color-toast-positive-subtle-border i-text-$color-toast-positive-text',
-  toastWarning: 'i-bg-$color-toast-warning-background i-border-$color-toast-warning-subtle-border i-text-$color-toast-warning-text',
-  toastNegative: 'i-bg-$color-toast-negative-background i-border-$color-toast-negative-subtle-border i-text-$color-toast-negative-text',
-  toastIcon: 'flex-shrink-0 rounded-full w-[16px] h-[16px] m-[8px] s-text-inverted',
-  toastIconPositive: 'i-text-$color-toast-positive-icon',
-  toastIconWarning: 'i-text-$color-toast-warning-icon',
-  toastIconNegative: 'i-bg-$color-toast-negative-icon',
-  toastIconLoading: 'animate-bounce',
-  toastContent: 'self-center mr-8 py-4 last-child:mb-0',
-  toastClose: 'ml-auto p-[8px]',
+  positive: 'i-bg-$color-toast-positive-background i-border-$color-toast-positive-subtle-border i-text-$color-toast-positive-text',
+  warning: 'i-bg-$color-toast-warning-background i-border-$color-toast-warning-subtle-border i-text-$color-toast-warning-text',
+  negative: 'i-bg-$color-toast-negative-background i-border-$color-toast-negative-subtle-border i-text-$color-toast-negative-text',
+  icon: 'flex-shrink-0 rounded-full w-[16px] h-[16px] m-[8px] text-white',
```

Comment:
If you merge latest changes from component-classes `text-white` was replaced with `s-text-inverted`

Reply:
fixed!